<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParametersWizard</class>
 <widget class="QDialog" name="ParametersWizard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1031</width>
    <height>564</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="0,1">
   <item row="1" column="0">
    <widget class="QTableView" name="parametersTable"/>
   </item>
   <item row="1" column="1">
    <widget class="QTabWidget" name="initialValuesTab"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="styleSheet">
      <string notr="true">font-size: 24px;
font-weight: bold;</string>
     </property>
     <property name="text">
      <string>Model Parameters Wizard</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QDialogButtonBox" name="closeButtons">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
